docs/diagrams/QuorumProposalRecv.md

```diff
@@ -0,0 +1,9 @@
+## QuorumProposalRecv
+
+![QuorumProposalRecv](/docs/diagrams/images/HotShotFlow-QuorumProposalRecv.drawio.png "QuorumProposalRecv")
+
+#### Notes
+* We emit a `ViewChange` event if we've seen evidence for a view change *and* if the quorum certificate on the proposal is valid.  We could optionally emit a `ViewChange` event if we've seen evidence but the quorum certificate on the proposal is invalid, but that is not the choice we make here.  Either choice is correct, but the former is simpler.  
+* We include a view sync certificate as view change evidence.  This isn't strictly required, but it makes the view change logic more understandable and avoids an attack where a view sync relay only sends a commit certificate to the next leader, resulting in other nodes rejecting the leader's proposal because it doesn't have proper view change evidence. It also signals nodes to handle view sync certificate evidence, which is handled differently than timeout certificate evidence or quorum certificate evidence. 
```

Comment:
Based on the diagram, the view sync commit cert isn't required as the view change evidence, unless the view doesn't match and there's no TC, but here it seems that the view sync cert is always included. Is the view sync cert different from the commit cert (because the former represents the "finalize" stage rather than "commit")? If so, shouldn't they all be included in the diagram?

I guess I'm confused about this, and so don't quite understand the attack described here as well.

Reply:
I will work on rewording this!  Thank you for the feedback!